src/VisualStudio/Core/Test.Next/Services/RemoteLanguageServerTests.cs

```diff
@@ -1,142 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-#nullable disable
-
-using System;
-using System.IO;
-using System.Linq;
-using System.Threading;
-using System.Threading.Tasks;
-using Microsoft.CodeAnalysis;
-using Microsoft.CodeAnalysis.Editor.UnitTests;
-using Microsoft.CodeAnalysis.Editor.UnitTests.Workspaces;
-using Microsoft.CodeAnalysis.LanguageServer.Handler;
-using Microsoft.CodeAnalysis.Remote;
-using Microsoft.CodeAnalysis.Remote.Testing;
-using Microsoft.CodeAnalysis.Test.Utilities;
-using Microsoft.VisualStudio.LanguageServer.Protocol;
-using Newtonsoft.Json.Linq;
-using Xunit;
-
-namespace Roslyn.VisualStudio.Next.UnitTests.Services
-{
-    [UseExportProvider]
-    [Trait(Traits.Feature, Traits.Features.RemoteHost)]
-    public class RemoteLanguageServerTests
```

Comment:
these tests were all present in workspacesymbolstests, except for a VB one which I added.  So it's safe to delete all these.